pybop/plotting/quick_plot.py

```diff
@@ -215,21 +215,22 @@ def quick_plot(params, cost, title="Scatter Plot", width=1024, height=576):
     model_output = cost.problem.evaluate(params)
     target_output = cost.problem.target()
 
-    # Create the figure using the StandardPlot class
-    fig = pybop.StandardPlot(
-        x=time_data,
-        y=model_output,
-        cost=cost,
-        y2=target_output,
-        xaxis_title="Time [s]",
-        yaxis_title=cost.problem.signal,
-        title=title,
-        trace_name="Model",
-        width=width,
-        height=height,
-    )()
-
-    # Display the figure
-    fig.show()
+    for i in range(0, cost.problem.n_outputs):
```

Comment:
Longer term, we might want to consider creating a subplot for this. Have both signals on a single plot would be nice.

Reply:
I agree! Something to look into